packages/playwright-test/src/common/transform.ts

```diff
@@ -238,20 +256,6 @@ export function wrapFunctionWithLocation<A extends any[], R>(func: (location: Lo
   };
 }
 
-// This will catch the playwright-test package as well
-const kPlaywrightInternalPrefix = path.resolve(__dirname, '../../../playwright');
```

Comment:
I'm not sure how this was supposed to work in production.

Reply:
I think this is for dev mode only, because usually any playwright* package is under `node_modules`.

Reply:
Ah, makes sense.